deployment/helm/node-feature-discovery/templates/serviceaccount.yaml

```
@@ -27,6 +27,21 @@ metadata:
   {{- end }}
 {{- end }}
 
+---
+{{- if and (eq .Values.topologyGC.enable true) (eq .Values.topologyGC.serviceAccount.create true) (eq .Values.topologyUpdater.enable true) }}
```

nit: could be simplified

```suggestion
{{- if and .Values.topologyGC.enable .Values.topologyGC.serviceAccount.create .Values.topologyUpdater.enable }}
```

---

done